kolibri/core/assets/src/views/ContentIcon.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 
-  <span>
-    <UiIcon ref="type-icon">
+  <div>
```

Comment:
Making this a div breaks the layout of rows on the import/export page.

![screen shot 2018-11-07 at 5 36 54 pm](https://user-images.githubusercontent.com/10248067/48172190-bf38e380-e2b3-11e8-8c3b-cae47c210f0f.png)